src/gui/window_data.py

```diff
@@ -42,6 +42,15 @@ class WindowData:
         margin=1,
     )
 
+    buttons = arcade.load_spritesheet(
+        "./assets/sprites/buttons.png",
+        sprite_height=16,
+        sprite_width=48,
+        columns=6,
+        count=24,
+        margin=0,
+    )
+

```

Comment:
I think we have strained the `WindowData` abstraction to breaking point. The sooner we break it up, the less painful it will be